### PR TITLE
chore(client): increase the api num as the latest server commit + 10

### LINKED ISSUE #545: [Bug] Api version of client is outdated
### Bug Type (问题类型)

exception / error (异常报错)

### Before submit

- [X] I had searched in the [issues](https://github.com/apache/hugegraph-toolchain/issues) and found no similar issues.

### Environment (环境信息)

- Server Version: v1.5.0
- Client Version: v1.0.0

### Expected & Actual behavior (期望与实际表现)

img_v3_025q_048c7ab3-5819-4019-bbde-3d27d9d5b0cg
The api version mismatch between client and server.

The bug is caused by the update of api version in server:
[    public static final Version VERSION = Version.of(ApiVersion.class, "0.71");](https://github.com/apache/incubator-hugegraph/blob/20d1e5228e39bd974079891fc713574fb14798c5/hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/version/ApiVersion.java#L125)
image
While the api version is between 0.38-0.70 in client:
7ae957869e71698f16587182d260b64

### Vertex/Edge example (问题点 / 边数据举例)

_No response_

### Schema [VertexLabel, EdgeLabel, IndexLabel] (元数据结构)

_No response_

### PR DESCRIPTION
**Pull request by aroundabout**
## Purpose of the PR

- close #545 

## Main Changes

1. The right range of api version is set to 0.81 (0.71+10)
2. Build client in dockerfile instead of using client:1.0.0

## Verifying these changes

tested by this yaml, where dandelionivy/hubble is my own repo
```yaml
version: '3'
services:
  server:
    image: hugegraph/hugegraph
    #image: dandelionivy/server
    container_name: graph
    ports:
      - 18081:8080

  hubble:
    #image: hugegraph/hubble
    image: dandelionivy/hubble
    container_name: hubble
    ports:
      - 8088:8088
```

now we can connect to the graph:
image

- [ ] Trivial rework / code cleanup without any test coverage. (No Need)
- [ ] Already covered by existing tests, such as *(please modify tests here)*.
- [ ] Need tests and can be verified as follows:
    - xxx

## Does this PR potentially affect the following parts?

- [x]  Nope
- [ ]  Dependencies (add/update license info) 
- [ ]  Modify configurations
- [ ]  The public API
- [ ]  Other affects (typed here)

## Documentation Status

- [ ]  `Doc - TODO` 
- [ ]  `Doc - Done` 
- [x]  `Doc - No Need`

### CODE REVIEW

**Comment by liuxiaocs7** [comment]
Note: now the api version check range is: `[0.38, 0.81)`

**Review by simon824 — DISMISSED** [review]

File: hugegraph-client/src/main/java/org/apache/hugegraph/driver/HugeClient.java [hunk 1]
```diff
@@ -109,7 +109,9 @@ private void initManagers(RestClient client, String graph) {
 
     private void checkServerApiVersion() {
         VersionUtil.Version apiVersion = VersionUtil.Version.of(this.version.getApiVersion());
-        VersionUtil.check(apiVersion, "0.38", "0.70", "hugegraph-api in server");
+        # TODO: find a way to keep the range of api version correct automatically
+        # 0.81 equals to the {latest_api_version} +10
```
Comment (liuxiaocs7) [inline hugegraph-client/src/main/java/org/apache/hugegraph/driver/HugeClient.java:113]: Line comments in java code should be `//` rather than `#`?

Comment (aroundabout) [inline hugegraph-client/src/main/java/org/apache/hugegraph/driver/HugeClient.java:113]:
```suggestion
        // TODO: find a way to keep the range of api version correct automatically
        // 0.81 equals to the {latest_api_version} +10  
```

Comment (aroundabout) [reply hugegraph-client/src/main/java/org/apache/hugegraph/driver/HugeClient.java]: Sorry, I made a mistake. It was a simple error.

**Review by imbajin — APPROVED** [review]

**Review by simon824 — APPROVED** [review]

**Pull request merged**